setup.py

```diff
@@ -33,7 +33,7 @@ def read(filename):
     "decorator < 5.0.0",  # 5.0.0 and later drops py2 support (transitive dep from pinject)
     "six == 1.12.0",
     "dnspython == 1.16.0",
-    "k8s == 0.17.0",
+    "k8s == 0.20.0",
```

Comment:
since v0.21.0 was just released, do we maybe want to use that?

Reply:
Not needed for this pr. But we'll be going for .21 for CRD support

Reply:
I think it is fine to upgrade to k8s 0.21.0 when taking the new models for ingress and crd into use